packages/ember-routing/lib/system/router.js

```diff
@@ -32,6 +32,18 @@ function K() { return this; }
 
 var slice = [].slice;
 
+
+var handledErrors = {};
```

Comment:
Would it be possible to use an object on the router instance instead of closure scope?